xlators/mgmt/glusterd/src/glusterd-utils.c

```diff
@@ -6118,7 +6143,7 @@ attach_brick_callback(struct rpc_req *req, struct iovec *iov, int count,
 int
 send_attach_req(xlator_t *this, struct rpc_clnt *rpc, char *path,
                 glusterd_brickinfo_t *brickinfo,
-                glusterd_brickinfo_t *other_brick, int op)
+                glusterd_brickinfo_t *other_brick, int op, int graceful_cleanup)
```

Comment:
The argument we pass here is actually a boolean. It's better to use the same type.

```suggestion
                glusterd_brickinfo_t *other_brick, int op, gf_boolean_t graceful_cleanup)
```